test/compiler/flowgraph/views_sourceinfo.spec.js

```diff
@@ -54,4 +59,6 @@ describe('Views get info on source time bounds', function() {
     test('(read stochastic -source "cdn" -from :' + from.valueOf() + ':; read stochastic -last :' + last.valueOf() + ': -source "cdn" ) | view view ',
          [[{from: from, to: null, last: null}, {from: null, to: null, last: last}]]);
 
+    test('(read stochastic -source "cdn" -last :' + last.valueOf() + ':; emit -last :' + last.valueOf() + ': ) | view view ',
+         [[{from: null, to: null, last: last}, {from: null, to: null, last: last}]]);
```

Comment:
I’d say this test is slightly redundant and I’d remove it. These test seem to exercise behavior with different graph topologies (not specific sources) and we already established `emit` works in the tests above.